src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/MetamaskFlow/index.js

```diff
@@ -207,7 +207,10 @@ class MetamaskFlow extends Component {
     return this.PaymentInfoCardData.find(el => el.title === "Channel Balance").value;
   };
 
-  shouldContinueBeEnabled = () => this.state.mpeBal > 0 && this.props.isServiceAvailable;
+  shouldContinueBeEnabled = () => {
+    const { mpeBal, totalPrice } = this.state;
+    return mpeBal >= totalPrice;
```

Comment:
this.props.isServiceAvailable; should also be considered..
--
And what happens if there is no balance in mpe but there is enough balance in channel?
Will continue button be enabled?

Reply:
Fix in the latest commit.